apps/charterafrica/src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,92 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import Section from "@/commons-ui/core/Section";
+import { Link } from "@commons-ui/next";
+import { Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import BackgroundBox from "@/charterafrica/components/BackgroundBox";
+import RichText from "@/charterafrica/components/RichText";
+import mapLinkTypeToHref from "@/charterafrica/payload/utils/mapLinkTypeToHref";
+
+const ErrorPage = React.forwardRef(function Error(props, ref) {
+  const {
+    statusCode,
+    description = [],
+    title = "",
+    createdAt,
+    updatedAt,
+    action,
+    ...other
+  } = props;
+
+  const link =
+    action &&
+    mapLinkTypeToHref({
```

Comment:
Can't think of a single reason why we would need this **_on_** front-end.

Reply:
Could we be forced to use REST API instead of payload import?🤔.

Because footer and header needs data from payload.

Reply:
We may have to but I'm hoping there is a fix for our current situation @koechkevin ... local APIs are just too nice.